ci/packet/packet-cluster.lokocfg.envsubst

```diff
@@ -137,6 +141,22 @@ component "rook" {
 
 component "flatcar-linux-update-operator" {}
 
+# openebs-storage-class component should always be the last to be installed
```

Comment:
Why is this here?

Reply:
This commit and the previous commits were done you,  I have not touched upon this commit.

Reply:
Well, in any case, it should probably not be there, as it is not related to the kube-apiserver extra flags.